core/src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/DsdeHttpDAO.scala

```diff
@@ -43,11 +43,18 @@ trait DsdeHttpDAO extends LazyLogging {
 
   protected def pipeline[A](implicit um: Unmarshaller[ResponseEntity, A]) = executeRequest[A] _
 
-  protected def when500(throwable: Throwable ): Boolean = {
-    throwable match {
-      case t: RawlsExceptionWithErrorReport => t.errorReport.statusCode.exists(_.intValue/100 == 5)
-      case _ => false
-    }
+  protected def when5xx(throwable: Throwable ): Boolean = DsdeHttpDAO.when5xx(throwable)
+
+
+}
+
+object DsdeHttpDAO {
```

Comment:
I don't know what "Dsde" and "DAO" stand for. Perhaps rename this with more a more explicit name, or add documentation?

Reply:
Sigh, I now see that this is in a file named "DsdeHttpDAO". So renaming is probably out of the question, but some documentation on the trait would be nice explaining what this is (and remove "Created by dvoet on 9/2/15.").

Reply:
DSDE is an [old organizational unit](https://docs.google.com/spreadsheets/d/1wn_XFAf54USob1UMqbk9SkxBgDXUV7UoY3fr24LVLeY/edit#gid=0) "Data Sciences and Data Engineering" that has since rolled up into DSP, that can definitely go away. [DAO](https://en.wikipedia.org/wiki/Data_access_object) is an at-least somewhat standard pattern.

Reply:
Yeah so this change is adding a [companion object](https://docs.scala-lang.org/overviews/scala-book/companion-objects.html) to this pre-existing `DsdeHttpDAO` class. I'll try to write my own personal interpretation of what this class is for, would welcome corrections though

Reply:
Thanks!